pyproject.toml

```diff
@@ -33,6 +33,7 @@ addopts = [
 ]
 markers = [
     "standalone: mark a test as standalone",
+    'large_mem: needs a lot of gpu memory (select/deselect with -m large_mem / -m "not large_mem")',
```

Comment:
Does this mean that only one model/test is running at a time?
as large memory could have different parameters... also does this apply to GPU or also CPU?

Reply:
Currently just GPU and currently we halve the number of pythread threads running these tests.